### PR TITLE
Fix wiki-link cli option.

### LINKED ISSUE #586: wiki deleted?
wiki links like https://github.com/troessner/reek/wiki/Duplicate-Method-Call are redirecting to the main github repo - did the wiki get deleted?  if so that's a great shame :-( or is it just moved to a new home ...?

### PR DESCRIPTION
**Pull request by troessner**
Fixes #586

### CODE REVIEW

File: features/reports/reports.feature [hunk 1]
```diff
@@ -183,16 +183,16 @@ Feature: Correctly formatted reports
       | smelly/ |
       | smelly  |
 
-  Scenario Outline: -U or --wiki-links adds helpful links to smell warnings
+  Scenario Outline: -U or --blob/master/docs-links adds helpful links to smell warnings
```
Comment (chastell) [inline features/reports/reports.feature]: I wish we had this switch – alas: we don’t.

File: features/reports/reports.feature [hunk 2]
```diff
@@ -200,16 +200,16 @@ Feature: Correctly formatted reports
       | -U               |
       | --wiki-links  |
 
-  Scenario Outline: --wiki-links is independent of --line-numbers
+  Scenario Outline: --blob/master/docs-links is independent of --line-numbers
```
Comment (chastell) [inline features/reports/reports.feature]: S(h)ame here.

**Comment by troessner** [comment]
>> S(h)ame here.

Yes, this is exactly what happens when you do a sloppy search-and-replace including an even sloppier "git add -p". Thanks for spotting this, fixed.

**Comment by chastell** [comment]
\o/ Merging!

**Pull request merged**